build-aws.ps1

```diff
@@ -43,9 +43,9 @@ Remove-OldLogsBeforeNewRun
 if(-not $SkipPester) {
   Write-Output "##teamcity[blockOpened name='Pester tests']"
   Write-Output "Importing Pester module"
-  Test-PowershellModuleInstalled "Pester" "4.9.0"
-  Test-PowershellModuleInstalled "PSScriptAnalyzer" "1.18.3"
-  Import-Module Pester -force
+  Import-PowerShellModule -Name "Pester" -MinimumVersion "4.9.0"
+  Import-PowerShellModule -Name "PSScriptAnalyzer" -MinimumVersion "1.18.3"
```

Comment:
This script wasn't installing PSScriptAnalyzer - it happens over https://github.com/OctopusDeploy/OctopusDSC/blob/master/Tests/OctopusDSC.Tests.ps1#L2.
Not sure how the duplicate import will affect things?

(I dont want to remove the import from OctopusDSC.Tests.ps1 - that allows us to just run `invoke-pester` in the folder and run the tests without trying to launch everything)

Reply:
What about keeping it here, but reference the new function?  Does this allow for the import of the helper file?

Reply:
Sounds good. That file can import `powershell-helpers.ps1` 👍 

to test it works, check you can:
* run `build-aws.ps1`
* run `invoke-pester OctopusDSC.Tests.ps1`  from the `Tests` directory

Reply:
I had to run it this way, Invoke-Pester .\Tests\OctopusDSC.Tests.ps1
The first line of OctopusDSC.Tests.ps1 is . Tests/powershell-helpers.ps1, so running it from the Tests directory failed since it was looking for another tests sub-folder.

Reply:
In which case, the `. Tests/powershell-helpers.ps1` needs to use `$PSScriptRoot`... 
Something along the lines of
```
$path = Join-Path -path $PSScriptRoot -childpath "powershell-helpers.ps1"
. $path
```

That way it will run from any folder

or similar

Reply:
Gonna have to remember that trick, thanks!